### ISSUE TITLE
Releasing prediction logits for NLP tasks?

### ISSUE THREAD

**Issue by tholiao** [facebookresearch/metaseq#25]
Hi,
Will the prediction logits for the 16 NLP tasks (Appendix A, Figure 6) be released?

**Comment by stephenroller** [comment]
I hadn't planned on it but I might be able to make that happen. There's a wider move to open source our eval suite that is underway.

**Comment by tholiao** [comment]
Thanks Stephen! This would be invaluable for fine-grained evaluation of these models and comparing against other LLMs.
